TestProject/Data/testData.json

```diff
@@ -0,0 +1,41 @@
+{
+  "loginData": {
+    "email": "ebageorgegp@gmail.com",
+    "password": "Eba@123",
+    "loginUrl": "http://localhost:5000/"
+  },
+  "createEducation": {
```

Comment:
can you please create a json file per test! Test data for test is never shared between tests